misc-utils/lsclocks.1.adoc

```diff
@@ -48,6 +48,9 @@ Do not try to discover dynamic clocks.
 *-d*, *--dynamic-clock* _path_
 Also display specified dynamic clock.
 
+*-c*, *--cpu-clock* _pid_
+Also display CPU clock of specified process.
```

Comment:
There is no info (on the man page and --help) that the option can be specified more than once.

Reply:
Added